CHANGES.rst

```diff
@@ -1,25 +1,47 @@
 Changes
 =======
 
-Version master (UNRELEASED)
----------------------------
-
-- Adds possibility to install REANA in different namespaces than default.
-- Moves to Helm deployment.
-- Adds command to bump common packages versions.
-- Supports prefixing by Helm release name.
-- Adds Helm Chart repo site.
-- Adds new ``python-unit-tests`` command.
-- Moves part of the documentation to docs.reana.io.
-- Adds Chart Release GitHub action.
-- Drops Python 2 support.
-- Adds email notificator service.
-- Moves database initialisation and admin creation after Helm installation.
-- Adds cronjob to send periodical status reports.
-- Adds announcement configmap configuration to display on the UI.
-- Adds Black formatter support.
-- Stops exposing unused Invenio-Accounts views.
-- Starts up Postgres DB container to run tests for componentes installing reana-db.
+Version 0.7.0 (2020-10-19)
+--------------------------
+
+- Users:
+    - Adds option to specify unpacked Docker images as workflow step requirement.
+    - Adds option to specify Kubernetes UID for jobs.
+    - Adds support for VOMS proxy as a new authentication method.
+    - Adds support for pulling private images using image pull secrets.
+    - Adds user profile page to the Web interface.
+    - Adds pagination on the workflow list web page.
+    - Adds option to ``logs`` command to filter job logs according to compute backend, docker image, status and step name.
+    - Adds new ``restart`` command to restart previously run or failed workflows.
+    - Adds option to specify operational options in the ``reana.yaml`` of the workflow.
+    - Fixes file preview functionality experience to allow/disallow certain file formats.
+    - Fixes workflow specification display to show runtime parameters.
+    - Changes CERN HTCondor compute backend to use the new ``myschedd`` connection library.
+    - Changes CERN Slurm compute backend to improve job status detection.
+    - Changes documentation to move large parts to docs.reana.io.
+    - Changes ``du`` command output format.
+    - Changes file loading to optimise CLI performance.
+    - Changes ``logs`` command to enhance formatting using marks and colours.
+    - Changes from Bravado to requests to improve download performance.
+    - Changes ``ping`` command to perform user access token validation.
+    - Changes defaults to accepts both ``reana.yaml`` and ``reana.yml`` filenames.
+    - Changes ``diff`` command to improve output formatting.
+- Administrators:
+    - Adds Helm chart and switches to Helm-based deployment technique instead of using now-deprecated ``reana-cluster``.
+    - Adds email notification service to inform administrators about system health.
+    - Adds announcement configuration option configuration to display any desired text on the Web UI.
```

Comment:
It is not clear to me when we use the capital letters or not (see https://github.com/reanahub/reana/pull/426/files#r508640338 and https://github.com/reanahub/reana/pull/426/files#diff-ff3c479edefad986d2fe6fe7ead575a46b086e3bbcf0ccc86d85efc4a4c63c79R13) so maybe the initial comment didn't make sense...

Reply:
Yeah, both are kind of possible... downcased to `web UI`.